sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/google-api.yaml

```diff
@@ -14,8 +14,12 @@ provider:
       metadata:
         name: Google API Keys
       detection:
+        pre:
+          reject:
+            content:
+              - "firebase"
         matching:
-          pattern: "\\b(AIza[\\w\\-]{35})\\b"
+          pattern: "\\b(AIza[0-9A-Za-z_\\-]{35})\\b"
```

Comment:
Hey, small FYI: \w is actually the same as what you replaced it with (even the _). Did you change it for clarity ?

Reply:
Yes. I have no issue replacing `[A-Za-z0-9_]` with `\w` but `[\\w\\-]` really looks weird to me. Moreover, I feel that `\w` as a meaning that does not fit the use here.
If you think we would be better reducing, fine by me.

Reply:
No totally fine by me also 👍